moma_sensor_tools/launch/handeye_calibration/easy_handeye.launch

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" ?>
+<launch>
+
+    <!-- Setting calibration namespace -->
+    <arg name="eye_on_hand" value="true"/>
+
+    <!-- The prefix of the namespace the node will run in, and of the folder in which the result will be saved" -->
+    <arg name="namespace_prefix" default="easy_handeye"/> 
+    
+    <!-- The namespace the node will run in, and the folder in which the result will be saved  -->
+    <arg name="namespace" value="$(arg namespace_prefix)_eye_on_hand" if="$(arg eye_on_hand)" />
+
+    <!-- The input reference frames -->
+    <arg name="robot_base_frame"        default="base_link" />
+    <arg name="robot_effector_frame"    default="panda_link8" />
+    <arg name="tracking_base_frame"     default="heand_eye_color_optical_frame" />
+    <arg name="tracking_marker_frame"   default="tag_2" />
+    
+    <!-- If true, a dummy calibration will be published to keep all frames in a single tf tree, hence visualized in RViz -->
+    <arg name="publish_dummy" default="true"/>
+    
+    <!-- GUI arguments -->
+    <!-- if true, rviz will be started with the provided config file-->
+    <arg name="start_rviz" default="true"/>
+    <arg name="rviz_config_file" default="$(find easy_handeye)/launch/rviz_easy_handeye.config"/>
+
+    <arg name="start_sampling_gui" default="true"/>
+    <arg name="rqt_perspective_file" default="$(find easy_handeye)/launch/rqt_easy_handeye.perspective"/>
+    
+    <!-- Dummy calibration to have a fully connected tf tree and see all frames -->
+    <node if="$(arg publish_dummy)" name="dummy_handeye" pkg="tf" type="static_transform_publisher" 
+        args="0 0 0.05 0 0 0 $(arg robot_effector_frame) $(arg tracking_base_frame) 10" />
+
+    <group ns="$(arg namespace)">
+
+        <!-- Store the parameters in the calibration's namespace-->
+        <param name="eye_on_hand" value="$(arg eye_on_hand)"/>
+        <param name="robot_base_frame" value="$(arg robot_base_frame)" />
+        <param name="robot_effector_frame" value="$(arg robot_effector_frame)" />
+        <param name="tracking_base_frame" value="$(arg tracking_base_frame)" />
+        <param name="tracking_marker_frame" value="$(arg tracking_marker_frame)" />
+
+        <param name="move_group"                value="panda_arm"/>
+        <param name="move_group_namespace"      value="/"/>
+        <param name="freehand_robot_movement"     value="true" />
+        <arg name="robot_velocity_scaling"      value="0.5" />
+        <arg name="robot_acceleration_scaling"  value="0.2" />
```

Comment:
As far as I remember these params are moveit related but it could be that were needed, otherwise the node would crash. We should add a comment